app/uk/gov/hmrc/helptosavefrontend/config/ErrorHandler.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.helptosavefrontend.config
+
+import javax.inject.{Inject, Singleton}
+
+import play.api.i18n.MessagesApi
+import play.api.mvc.Request
+import play.twirl.api.Html
+import uk.gov.hmrc.helptosavefrontend.models.HtsContext
+import uk.gov.hmrc.helptosavefrontend.views
+import uk.gov.hmrc.play.bootstrap.http.FrontendErrorHandler
+
+@Singleton
+class ErrorHandler @Inject() (implicit val messagesApi: MessagesApi, val appConfig: FrontendAppConfig) extends FrontendErrorHandler {
+  override def standardErrorTemplate(pageTitle: String, heading: String, message: String)(implicit request: Request[_]): Html = {
+    implicit val htsContext: HtsContext = HtsContext(true)
```

Comment:
this hacky bit is inherited from existing code `implicit val htsContext: HtsContext = HtsContext(authorised = false)` from the removed `FrontendGloal` class except that i used `true` flag above